docs/NEW_GUIDE_DRAFT.en.md

```diff
@@ -0,0 +1,68 @@
+`NEAR CLI Core commands`
+login
+create-account
+    - implicit
+    - subaccount
+<!-- account Id is entered in this step -->
+manage-account
+    - keys
+        - view
+        - add
+        - delete
+        - nonce
+    - local-keys
+        <!-- Needs to be designed (with multikey management) -->
+    - contract
+        - deploy
+        - dev-deploy
+        - state
+        - wasm-hash
+        - view-call
+        - change-call
+    - state
+    - delete
+transfer
+<!-- TODO: feels a bit out of place, but definitely important. Maybe we need to add other view functions and have 1 subcommand for them. Or it can be one of the helpers-->
+transaction-status
```

Comment:
Maybe we should have some sort of basic explorer command to view transaction status, epoch status, etc.

Reply:
Yes! We were planning explorer as an extension.

Reply:
I feel I have never had a need to view transaction status from CLI unless I am wearing a smart-contract/blockchain developer hat. If I initiated a transaction from CLI, I will get the report from CLI about the transaction status, so I don't need to view the transaction status manually day-to-day, thus according to our core CLI definition, it should not be part of the near-cli-rs. On the other hand, it could be a small helper, so we may just land it under helpers

Reply:
I gues we can get move `get-recent-block-hash` to `explorer`or `developer` extension too.